syn/python/get_kge.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python3
+
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Parse a yosys area report and give a kGE equivalient
+
+import re
+import argparse
+
+weighted_dict = {}
+
+
+def read_lib(lib_file_path):
+    with open(lib_file_path, 'r') as f:
+        lib_file = f.readlines()
+
+    cell_dict = {}
+    cell_name = ''
+    for line in lib_file:
+        if re.search(r'^  cell \(', line):
```

Comment:
As a general note, `re.search(r'^foo')` is the same thing as `re.match(r'foo')`. Probably cleaner to use that explicitly? And, in this case, you have a literal string. So you could just write
```python
if line.startswith('    cell ('):
    ...
```
which I think is easier to understand.

Reply:
Yep will do